packages/rocketchat-ldap/config_server.coffee

```diff
@@ -2,11 +2,13 @@ MeteorWrapperLdapjs = Npm.require 'ldapjs'
 
 Meteor.startup ->
 	RocketChat.settings.addGroup 'LDAP'
-	RocketChat.settings.add 'LDAP_Enable'     , false    , { type: 'boolean', group: 'LDAP', public: true }
-	RocketChat.settings.add 'LDAP_Url'        , 'ldap://', { type: 'string' , group: 'LDAP' }
-	RocketChat.settings.add 'LDAP_Port'       , '389'    , { type: 'string' , group: 'LDAP' }
-	RocketChat.settings.add 'LDAP_DN'         , ''       , { type: 'string' , group: 'LDAP', i18nLabel: 'LDAP_Dn', public: true }
-	RocketChat.settings.add 'LDAP_Bind_Search', ''       , { type: 'string' , group: 'LDAP' }
+	RocketChat.settings.add 'LDAP_Enable', false, { type: 'boolean', group: 'LDAP', public: true }
+	RocketChat.settings.add 'LDAP_Url', 'ldap://', { type: 'string' , group: 'LDAP' }
+	RocketChat.settings.add 'LDAP_Port', '389', { type: 'string' , group: 'LDAP' }
+	RocketChat.settings.add 'LDAP_DN', '', { type: 'string' , group: 'LDAP', i18nLabel: 'LDAP_Dn', public: true }
+	RocketChat.settings.add 'LDAP_Bind_Search', '', { type: 'string' , group: 'LDAP' }
+	RocketChat.settings.add 'LDAP_Sync_User_Data', false, { type: 'boolean' , group: 'LDAP' }
+	RocketChat.settings.add 'LDAP_Sync_User_Data_FieldMap', '{"cn":"name", "mail":"email"}', { type: 'string' , group: 'LDAP' }
```

Comment:
As a side note, as people keep adding more and more settings, you may want to start adding some sort of shorthand for referring to them rather than raw strings. It could potentially even throw an exception if you try to call a `get` on an unknown setting.

I'm not that familiar with the code / with Meteor, but just my two :dollar: cents